src/lib/PetitioResponse.ts

```diff
@@ -63,9 +63,11 @@ export class PetitioResponse {
 	}
 
 	/**
-	 * @return {*} The response body decoded as a UTF-8 string from the buffer.
+	 * @param {BufferEncoding} [encoding="utf8"] The encoding to use.
+	 * @return {string} The response body decoded as a UTF-8 (or whatever specified in `encoding`) string from the buffer.
```

Comment:
```suggestion
	 * @return {string} The response body decoded as as a string from the buffer, using either the encoding specified in `encoding` or UTF-8 by default..
```
Just nitpicking for formality